core/src/com/unciv/logic/HexMath.kt

```diff
@@ -59,6 +85,20 @@ object HexMath {
         return Vector2(cubicCoord.y, -cubicCoord.z)
     }
 
+    fun cubic2EvenQCoords(cubicCoord: Vector3): Vector2 {
+        return Vector2(cubicCoord.x, cubicCoord.z + (cubicCoord.x + (cubicCoord.x.toInt() and 1)) / 2)
+    }
+    fun evenQ2CubicCoords(evenQCoord: Vector2): Vector3 {
```

Comment:
Beg pardon, but what are evenQ coordinates?

Reply:
Sorry for not clearing this. It's a system of coordinates for rectangular hex maps where the hexagon coordinates are assigned as in figure below:
![evenq](https://user-images.githubusercontent.com/17114100/73777895-8aebf580-478a-11ea-9658-5a3f8d985a50.png)

In creating a rectangular map i think this coordinate system feels natural, so i generate the map using this coordinate system and then convert it to our Hex coordinate system.